openid-4-verifiable-presentations-1_0.md

```diff
@@ -699,13 +699,15 @@ Content-Type: application/x-www-form-urlencoded
   state=eyJhb...6-sVA
 ```
 
-If the Response URI has successfully processed the Authorization Response or Authorization Error Response, it MUST respond with HTTP status code 200.
+If the Response URI has successfully processed the Authorization Response or Authorization Error Response, it MUST respond with HTTP status code 200 and a JSON object.
```

Comment:
```suggestion
If the Response URI has successfully processed the Authorization Response or Authorization Error Response, it MUST respond with HTTP status code of 200 and a JSON object in the response body.
```

Reply:
Refined to include the text you'd suggested above:

```suggestion
If the Response URI has successfully processed the Authorization Response or Authorization Error Response, it MUST respond with an HTTP status code of 200 with `Content-Type` of `application/json` and a JSON object in the response body.
```